docs/source/specs/openapi.json

```
@@ -2247,6 +2297,55 @@
           }
         ]
       },
+      "RoleOutDynamic": {
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Role"
+          },
+          {
+            "$ref": "#/components/schemas/UUID"
+          },
+          {
+            "$ref": "#/components/schemas/Timestamped"
+          },
+          {
+            "type": "object",
+            "required": ["policyCount", "accessCount", "applications", "system", "platform_default"],
```

Formatting suggestion for consistency.
```suggestion
            "required": [
              "policyCount", 
              "accessCount", 
              "applications", 
              "system", 
              "platform_default"
            ],
```

---

👍